content/tutorials/generic/pro-oem-condition-monitoring-kit/pro-oem-condition-monitoring-kit.md

```diff
@@ -45,13 +45,13 @@ Additionally, you may need the following accessories:
 
 ## Getting Started with Your Kit
 
-1. Get to [cloud.arduino.cc](https://cloud.arduino.cc/), sign in, or Create an Arduino account. Rescue your Arduino IoT Cloud 3 months voucher code using this [Procedure](https://docs.arduino.cc/tutorials/generic/cloud-business-voucher-redeem) and [Get Started](https://docs.arduino.cc/arduino-cloud/getting-started/arduino-cloud-for-business!)
+1. Get to [cloud.arduino.cc](https://cloud.arduino.cc/), sign in, or Create an Arduino account. Rescue your Arduino IoT Cloud 3 months voucher code using this [Procedure](https://docs.arduino.cc/tutorials/generic/cloud-business-voucher-redeem) and [Get Started](https://docs.arduino.cc/arduino-cloud/getting-started/arduino-cloud-for-business)!
 
-2. Download [Arduino IDE](https://www.arduino.cc/en/software#future-version-of-the-arduino-ide) and follow the [Installation Procedure](https://docs.arduino.cc/software/ide-v2/tutorials/getting-started/ide-v2-downloading-and-installing) and [Get Started with it](https://docs.arduino.cc/software/ide-v2/tutorials/getting-started-ide-v2)
+2. Download [Arduino IDE](https://www.arduino.cc/en/software#future-version-of-the-arduino-ide) and follow the [Installation Procedure](https://docs.arduino.cc/software/ide-v2/tutorials/getting-started/ide-v2-downloading-and-installing) and [Get Started with it](https://docs.arduino.cc/software/ide-v2/tutorials/getting-started-ide-v2).
 
 3. Download [Arduino PLC IDE](https://www.arduino.cc/en/software#arduino-plc-ide) and follow the [Installation Procedure](https://docs.arduino.cc/software/plc-ide/tutorials/plc-ide-setup-license) and [Get Started with it](https://docs.arduino.cc/software/plc-ide/tutorials/plc-programming-introduction). Use the PLC KEY code supplied in the bundle to unlock for lifetime the Portenta Machine Control to be programmed with PLC IDE.
 
-4. Check the [Portenta Machine Control docs](https://docs.arduino.cc/hardware/portenta-machine-control) to retrieve the product resources. Carefully read the [datasheet](https://docs.arduino.cc/resources/datasheets/AKX00032-datasheet.pdf) and the [User Manual](). On [docs.arduino.cc](http://docs.arduino.cc) are also available the [Pinout](https://docs.arduino.cc/resources/pinouts/AKX00032-full-pinout.pdf) and [Schematics](https://docs.arduino.cc/resources/schematics/AKX00032-schematics.pdf).
+4. Check the [Portenta Machine Control docs](https://docs.arduino.cc/hardware/portenta-machine-control) to retrieve the product resources. Carefully read the [datasheet](https://docs.arduino.cc/resources/datasheets/AKX00032-datasheet.pdf) and the [User Manual](). On [docs.arduino.cc](http://docs.arduino.cc) the [Pinout](https://docs.arduino.cc/resources/pinouts/AKX00032-full-pinout.pdf) and [Schematics](https://docs.arduino.cc/resources/schematics/AKX00032-schematics.pdf) are also available.
```

Comment:
```suggestion
4. Check the [Portenta Machine Control docs](https://docs.arduino.cc/hardware/portenta-machine-control) to retrieve the product resources. Carefully read the [datasheet](https://docs.arduino.cc/resources/datasheets/AKX00032-datasheet.pdf). On [docs.arduino.cc](http://docs.arduino.cc) the [Pinout](https://docs.arduino.cc/resources/pinouts/AKX00032-full-pinout.pdf) and [Schematics](https://docs.arduino.cc/resources/schematics/AKX00032-schematics.pdf) are also available.
```

Reply:
@jcarolinares until we have the user manual for the Portenta Machine Control, we should remove this sentence. I spoke with Francesca, and she is ok with this as well.

Reply:
Indeed, thank you.